react/components/atoms/button/button.js

```diff
@@ -65,12 +95,13 @@ export class Button extends mix(PureComponent).with(IdentifiableMixin) {
                 {this.props.icon ? (
                     <Icon
                         icon={this.props.icon}
-                        color="#ffffff"
-                        style={styles.icon}
+                        color={this.props.iconColor || "#ffffff"}
+                        fill={this.props.iconFillColor || "#ffffff"}

```

Comment:
Couldn't you set `#ffffff` as the default prop value, like you did with the prop textColor?